pennylane/math/utils.py

```diff
@@ -400,3 +400,25 @@ def requires_grad(tensor, interface=None):
         return isinstance(tensor, jax.core.Tracer)
 
     raise ValueError(f"Argument {tensor} is an unknown object")
+
+
+def in_backprop(tensor, interface=None):

```

Comment:
Is this the same as `requires_grad` from this file (just no JAX and Torch part yet)?

Reply:
Not exactly, this was created to check if we are in the backprop environment (autograd and tf). The problem with `requires_grad()` is that it checks the trainability and not if we actually backpropagate.